code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -456,7 +456,7 @@
 	..()
 
 /datum/reagent/toxin/fakebeer //disguised as normal beer for use by emagged brobots
-	name = "Beer"
+	name = "Beer...?"
```

Comment:
do you think we can name this Fake Beer and on New() change the name to beer's name?

Reply:
It wouldn't fix https://github.com/tgstation/tgstation/issues/64569 (i've tried)

Reply:
A lot of stuff doesn't use `initial` for names